### PR TITLE
packages yum: add a missing release number

### PR DESCRIPTION
**Pull request by komainu8**
TODO: This is testing now.

Fix mroonga/mroonga#429. Reported by Yuutakasan. Thanks!!!

### CODE REVIEW

File: packages/yum/groonga.spec.in [hunk 1]
```diff
@@ -45,7 +45,7 @@ on relational data model.
 Summary:	Runtime libraries for Groonga
 Group:		System Environment/Libraries
 License:	LGPLv2 and (MIT or GPLv2)
-Requires:	arrow-libs = @APACHE_ARROW_VERSION@
+Requires:	arrow-libs = @APACHE_ARROW_VERSION@-%{release}
```
Comment (kou) [inline packages/yum/groonga.spec.in:48]: `arrow-libs` doesn't use the same release version as ours.

```suggestion
Requires:	arrow-libs = @APACHE_ARROW_VERSION@-1%{?dist}
```

We should change `BuildRequires:	arrow-devel = @APACHE_ARROW_VERSION@` too.

**Comment by komainu8** [comment]
We also need to update `vendor/apache-arrow-source`.
Because Groonga decide of requirment version from version of `vendor/apache-arrow-source`.

Apache Arrow version is 4.0.0 when we make the package for Groonga 11.0.4, but Apache Arrow version up in 19 June 2021.

```
$ sudo yum --enablerepo=local install groonga
Failed to set locale, defaulting to C
Loaded plugins: fastestmirror
Determining fastest mirrors
epel/x86_64/metalink                                                                                                        | 9.2 kB  00:00:00     
 * base: ftp.riken.jp
 * epel: nrt.edge.kernel.org
 * extras: ftp.riken.jp
 * updates: ftp.riken.jp
base                                                                                                                        | 3.6 kB  00:00:00     
epel                                                                                                                        | 4.7 kB  00:00:00     
extras                                                                                                                      | 2.9 kB  00:00:00     
local                                                                                                                       | 3.0 kB  00:00:00     
updates                                                                                                                     | 2.9 kB  00:00:00     
(1/8): base/7/x86_64/group_gz                                                                                               | 153 kB  00:00:00     
(2/8): epel/x86_64/group_gz                                                                                                 |  96 kB  00:00:00     
(3/8): epel/x86_64/updateinfo                                                                                               | 1.0 MB  00:00:00     
(4/8): extras/7/x86_64/primary_db                                                                                           | 242 kB  00:00:00     
(5/8): local/primary_db                                                                                                     |  10 kB  00:00:00     
(6/8): base/7/x86_64/primary_db                                                                                             | 6.1 MB  00:00:00     
(7/8): updates/7/x86_64/primary_db                                                                                          | 9.5 MB  00:00:00     
(8/8): epel/x86_64/primary_db                                                                                               | 6.9 MB  00:00:00     
Resolving Dependencies
--> Running transaction check
---> Package groonga.x86_64 0:11.0.3-1.el7 will be updated
---> Package groonga.x86_64 0:11.0.5-1.el7 will be an update
--> Processing Dependency: groonga-libs = 11.0.5-1.el7 for package: groonga-11.0.5-1.el7.x86_64
--> Processing Dependency: groonga-plugin-suggest = 11.0.5-1.el7 for package: groonga-11.0.5-1.el7.x86_64
--> Running transaction check
---> Package groonga-libs.x86_64 0:11.0.3-1.el7 will be updated
---> Package groonga-libs.x86_64 0:11.0.5-1.el7 will be an update
--> Processing Dependency: arrow-libs = 4.0.0-1.el7 for package: groonga-libs-11.0.5-1.el7.x86_64
---> Package groonga-plugin-suggest.x86_64 0:11.0.3-1.el7 will be updated
---> Package groonga-plugin-suggest.x86_64 0:11.0.5-1.el7 will be an update
--> Finished Dependency Resolution
Error: Package: groonga-libs-11.0.5-1.el7.x86_64 (local)
           Requires: arrow-libs = 4.0.0-1.el7
           Installed: arrow-libs-4.0.1-1.el7.x86_64 (@groonga-centos)
               arrow-libs = 4.0.1-1.el7
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
[vagrant@localhost ~]$ sudo yum remove arrow-libs
Failed to set locale, defaulting to C
Loaded plugins: fastestmirror
Resolving Dependencies
--> Running transaction check
---> Package arrow-libs.x86_64 0:4.0.1-1.el7 will be erased
--> Processing Dependency: libarrow.so.400()(64bit) for package: groonga-libs-11.0.3-1.el7.x86_64
--> Running transaction check
---> Package groonga-libs.x86_64 0:11.0.3-1.el7 will be erased
--> Processing Dependency: groonga-libs = 11.0.3-1.el7 for package: groonga-11.0.3-1.el7.x86_64
--> Processing Dependency: groonga-libs = 11.0.3-1.el7 for package: groonga-plugin-suggest-11.0.3-1.el7.x86_64
--> Processing Dependency: libgroonga.so.0()(64bit) for package: groonga-11.0.3-1.el7.x86_64
--> Processing Dependency: libgroonga.so.0()(64bit) for package: groonga-plugin-suggest-11.0.3-1.el7.x86_64
--> Running transaction check
---> Package groonga.x86_64 0:11.0.3-1.el7 will be erased
---> Package groonga-plugin-suggest.x86_64 0:11.0.3-1.el7 will be erased
--> Finished Dependency Resolution

Dependencies Resolved
```

**Comment by kou** [comment]
If there are newer Apache Arrow packages, Groonga package should use the specified Apache Arrow package. (4.0.0 for this case.)

**Comment by kou** [comment]
If Groonga package always requires the latest Apache Arrow package, this problem will be happen in the future again.

**Comment by komainu8** [comment]
Ah...
My environment is wrong.
I'm already installed arrow-libs-4.0.1 with groonga-11.0.3 into it.

Currently, we are installed arrow-libs-4.0.1 when we installed Groonga 11.0.3. Because Groonga 11.0.3 is not include https://github.com/groonga/groonga/pull/1199.

**Comment by kou** [comment]
Does it mean that #1199 fixes this?

**Comment by komainu8** [comment]
Probably.
We success install Groonga even if with out this PR.

```
$ sudo yum install -y --enablerepo=epel groonga
Failed to set locale, defaulting to C
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
epel/x86_64/metalink                                                                                                        | 5.4 kB  00:00:00     
 * base: ty1.mirror.newmediaexpress.com
 * epel: ftp.yz.yamagata-u.ac.jp
 * extras: ty1.mirror.newmediaexpress.com
 * updates: ty1.mirror.newmediaexpress.com
epel                                                                                                                        | 4.7 kB  00:00:00     
groonga-centos                                                                                                              | 3.0 kB  00:00:00     
(1/4): epel/x86_64/group_gz                                                                                                 |  96 kB  00:00:00     
(2/4): epel/x86_64/updateinfo                                                                                               | 1.0 MB  00:00:00     
(3/4): groonga-centos/7/x86_64/primary_db                                                                                   | 2.6 MB  00:00:00     
epel/x86_64/primary_db         FAILED                                          
https://epel.01link.hk/7/x86_64/repodata/974fd52c8b66c525d9c64b32b8b7b67519c9996ee3d0e8f85bcc23907ceb05a4-primary.sqlite.bz2: [Errno 14] HTTPS Error 404 - Not Found
Trying other mirror.
To address this issue please refer to the below wiki article 

https://wiki.centos.org/yum-errors

If above article doesn't help to resolve this issue please use https://bugs.centos.org/.

(4/4): epel/x86_64/primary_db                                                                                               | 6.9 MB  00:00:01     
Resolving Dependencies
--> Running transaction check
---> Package groonga.x86_64 0:11.0.4-1.el7 will be installed
--> Processing Dependency: groonga-libs = 11.0.4-1.el7 for package: groonga-11.0.4-1.el7.x86_64
--> Processing Dependency: groonga-plugin-suggest = 11.0.4-1.el7 for package: groonga-11.0.4-1.el7.x86_64
--> Processing Dependency: libgroonga.so.0()(64bit) for package: groonga-11.0.4-1.el7.x86_64
--> Processing Dependency: libmsgpackc.so.2()(64bit) for package: groonga-11.0.4-1.el7.x86_64
--> Running transaction check
---> Package groonga-libs.x86_64 0:11.0.4-1.el7 will be installed
--> Processing Dependency: arrow-libs = 4.0.0 for package: groonga-libs-11.0.4-1.el7.x86_64
--> Processing Dependency: libzstd for package: groonga-libs-11.0.4-1.el7.x86_64
--> Processing Dependency: libarrow.so.400()(64bit) for package: groonga-libs-11.0.4-1.el7.x86_64
--> Processing Dependency: libxxhash.so.0()(64bit) for package: groonga-libs-11.0.4-1.el7.x86_64
--> Processing Dependency: libzstd.so.1()(64bit) for package: groonga-libs-11.0.4-1.el7.x86_64
---> Package groonga-plugin-suggest.x86_64 0:11.0.4-1.el7 will be installed
--> Processing Dependency: libevent-2.0.so.5()(64bit) for package: groonga-plugin-suggest-11.0.4-1.el7.x86_64
--> Processing Dependency: libzmq.so.3()(64bit) for package: groonga-plugin-suggest-11.0.4-1.el7.x86_64
---> Package msgpack.x86_64 0:3.1.0-4.el7 will be installed
--> Running transaction check
---> Package arrow-libs.x86_64 0:4.0.0-1.el7 will be installed
--> Processing Dependency: boost169-filesystem for package: arrow-libs-4.0.0-1.el7.x86_64
--> Processing Dependency: boost169-system for package: arrow-libs-4.0.0-1.el7.x86_64
--> Processing Dependency: brotli for package: arrow-libs-4.0.0-1.el7.x86_64
--> Processing Dependency: gflags for package: arrow-libs-4.0.0-1.el7.x86_64
--> Processing Dependency: glog for package: arrow-libs-4.0.0-1.el7.x86_64
--> Processing Dependency: utf8proc for package: arrow-libs-4.0.0-1.el7.x86_64
--> Processing Dependency: libbrotlicommon.so.1()(64bit) for package: arrow-libs-4.0.0-1.el7.x86_64
--> Processing Dependency: libbrotlidec.so.1()(64bit) for package: arrow-libs-4.0.0-1.el7.x86_64
--> Processing Dependency: libbrotlienc.so.1()(64bit) for package: arrow-libs-4.0.0-1.el7.x86_64
--> Processing Dependency: libutf8proc.so.1()(64bit) for package: arrow-libs-4.0.0-1.el7.x86_64
---> Package libevent.x86_64 0:2.0.21-4.el7 will be installed
---> Package libzstd.x86_64 0:1.5.0-1.el7 will be installed
---> Package xxhash-libs.x86_64 0:0.8.0-3.el7 will be installed
---> Package zeromq3.x86_64 0:3.2.5-1.el7 will be installed
--> Processing Dependency: libpgm-5.2.so.0()(64bit) for package: zeromq3-3.2.5-1.el7.x86_64
--> Running transaction check
---> Package boost169-filesystem.x86_64 0:1.69.0-2.el7 will be installed
---> Package boost169-system.x86_64 0:1.69.0-2.el7 will be installed
---> Package brotli.x86_64 0:1.0.7-5.el7 will be installed
---> Package gflags.x86_64 0:2.1.1-6.el7 will be installed
---> Package glog.x86_64 0:0.3.3-8.el7 will be installed
---> Package openpgm.x86_64 0:5.2.122-2.el7 will be installed
---> Package utf8proc.x86_64 0:1.3.1-2.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

===================================================================================================================================================
 Package                                    Arch                       Version                            Repository                          Size
===================================================================================================================================================
Installing:
 groonga                                    x86_64                     11.0.4-1.el7                       groonga-centos                      71 k
Installing for dependencies:
 arrow-libs                                 x86_64                     4.0.0-1.el7                        groonga-centos                     3.5 M
 boost169-filesystem                        x86_64                     1.69.0-2.el7                       epel                                41 k
 boost169-system                            x86_64                     1.69.0-2.el7                       epel                               7.5 k
 brotli                                     x86_64                     1.0.7-5.el7                        epel                               318 k
 gflags                                     x86_64                     2.1.1-6.el7                        epel                                72 k
 glog                                       x86_64                     0.3.3-8.el7                        epel                                57 k
 groonga-libs                               x86_64                     11.0.4-1.el7                       groonga-centos                     2.3 M
 groonga-plugin-suggest                     x86_64                     11.0.4-1.el7                       groonga-centos                      52 k
 libevent                                   x86_64                     2.0.21-4.el7                       base                               214 k
 libzstd                                    x86_64                     1.5.0-1.el7                        epel                               370 k
 msgpack                                    x86_64                     3.1.0-4.el7                        epel                                27 k
 openpgm                                    x86_64                     5.2.122-2.el7                      epel                               171 k
 utf8proc                                   x86_64                     1.3.1-2.el7                        epel                                63 k
 xxhash-libs                                x86_64                     0.8.0-3.el7                        epel                                26 k
 zeromq3                                    x86_64                     3.2.5-1.el7                        epel                               344 k

Transaction Summary
===================================================================================================================================================
Install  1 Package (+15 Dependent packages)

Total download size: 7.6 M
Installed size: 36 M
Downloading packages:
warning: /var/cache/yum/x86_64/7/epel/packages/boost169-filesystem-1.69.0-2.el7.x86_64.rpm: Header V3 RSA/SHA256 Signature, key ID 352c64e5: NOKEY
Public key for boost169-filesystem-1.69.0-2.el7.x86_64.rpm is not installed
(1/16): boost169-filesystem-1.69.0-2.el7.x86_64.rpm                                                                         |  41 kB  00:00:00     
(2/16): boost169-system-1.69.0-2.el7.x86_64.rpm                                                                             | 7.5 kB  00:00:00     
(3/16): brotli-1.0.7-5.el7.x86_64.rpm                                                                                       | 318 kB  00:00:00     
warning: /var/cache/yum/x86_64/7/groonga-centos/packages/arrow-libs-4.0.0-1.el7.x86_64.rpm: Header V4 RSA/SHA256 Signature, key ID 34839225: NOKEY
Public key for arrow-libs-4.0.0-1.el7.x86_64.rpm is not installed
(4/16): arrow-libs-4.0.0-1.el7.x86_64.rpm                                                                                   | 3.5 MB  00:00:00     
(5/16): gflags-2.1.1-6.el7.x86_64.rpm                                                                                       |  72 kB  00:00:00     
(6/16): groonga-11.0.4-1.el7.x86_64.rpm                                                                                     |  71 kB  00:00:00     
(7/16): groonga-plugin-suggest-11.0.4-1.el7.x86_64.rpm                                                                      |  52 kB  00:00:00     
(8/16): glog-0.3.3-8.el7.x86_64.rpm                                                                                         |  57 kB  00:00:00     
(9/16): libevent-2.0.21-4.el7.x86_64.rpm                                                                                    | 214 kB  00:00:00     
(10/16): libzstd-1.5.0-1.el7.x86_64.rpm                                                                                     | 370 kB  00:00:00     
(11/16): msgpack-3.1.0-4.el7.x86_64.rpm                                                                                     |  27 kB  00:00:00     
(12/16): groonga-libs-11.0.4-1.el7.x86_64.rpm                                                                               | 2.3 MB  00:00:00     
(13/16): openpgm-5.2.122-2.el7.x86_64.rpm                                                                                   | 171 kB  00:00:00     
(14/16): utf8proc-1.3.1-2.el7.x86_64.rpm                                                                                    |  63 kB  00:00:00     
(15/16): xxhash-libs-0.8.0-3.el7.x86_64.rpm                                                                                 |  26 kB  00:00:00     
(16/16): zeromq3-3.2.5-1.el7.x86_64.rpm                                                                                     | 344 kB  00:00:00     
---------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                              3.3 MB/s | 7.6 MB  00:00:02     
Retrieving key from file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
Importing GPG key 0x352C64E5:
 Userid     : "Fedora EPEL (7) <epel@fedoraproject.org>"
 Fingerprint: 91e9 7d7c 4a5e 96f1 7f3e 888f 6a2f aea2 352c 64e5
 Package    : epel-release-7-11.noarch (@extras)
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
Retrieving key from file:///etc/pki/rpm-gpg/RPM-GPG-KEY-34839225
Importing GPG key 0x34839225:
 Userid     : "Groonga Key (Groonga Official Signing Key) <packages@groonga.org>"
 Fingerprint: 2701 f317 cfcc cb97 5cad e9c2 624c f774 3483 9225
 Package    : groonga-release-2020.1.29-1.noarch (@/groonga-release-latest.noarch)
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-34839225
Retrieving key from file:///etc/pki/rpm-gpg/RPM-GPG-KEY-45499429
Importing GPG key 0x45499429:
 Userid     : "groonga Key (groonga Official Signing Key) <packages@groonga.org>"
 Fingerprint: c97e 4649 a205 1d0c ea1a 73f9 72a7 496b 4549 9429
 Package    : groonga-release-2020.1.29-1.noarch (@/groonga-release-latest.noarch)
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-45499429
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : msgpack-3.1.0-4.el7.x86_64                                                                                                     1/16 
  Installing : boost169-system-1.69.0-2.el7.x86_64                                                                                            2/16 
  Installing : libzstd-1.5.0-1.el7.x86_64                                                                                                     3/16 
  Installing : gflags-2.1.1-6.el7.x86_64                                                                                                      4/16 
  Installing : glog-0.3.3-8.el7.x86_64                                                                                                        5/16 
  Installing : boost169-filesystem-1.69.0-2.el7.x86_64                                                                                        6/16 
  Installing : utf8proc-1.3.1-2.el7.x86_64                                                                                                    7/16 
  Installing : xxhash-libs-0.8.0-3.el7.x86_64                                                                                                 8/16 
  Installing : libevent-2.0.21-4.el7.x86_64                                                                                                   9/16 
  Installing : brotli-1.0.7-5.el7.x86_64                                                                                                     10/16 
  Installing : arrow-libs-4.0.0-1.el7.x86_64                                                                                                 11/16 
  Installing : groonga-libs-11.0.4-1.el7.x86_64                                                                                              12/16 
  Installing : openpgm-5.2.122-2.el7.x86_64                                                                                                  13/16 
  Installing : zeromq3-3.2.5-1.el7.x86_64                                                                                                    14/16 
  Installing : groonga-plugin-suggest-11.0.4-1.el7.x86_64                                                                                    15/16 
  Installing : groonga-11.0.4-1.el7.x86_64                                                                                                   16/16 
  Verifying  : openpgm-5.2.122-2.el7.x86_64                                                                                                   1/16 
  Verifying  : brotli-1.0.7-5.el7.x86_64                                                                                                      2/16 
  Verifying  : glog-0.3.3-8.el7.x86_64                                                                                                        3/16 
  Verifying  : gflags-2.1.1-6.el7.x86_64                                                                                                      4/16 
  Verifying  : boost169-filesystem-1.69.0-2.el7.x86_64                                                                                        5/16 
  Verifying  : libevent-2.0.21-4.el7.x86_64                                                                                                   6/16 
  Verifying  : libzstd-1.5.0-1.el7.x86_64                                                                                                     7/16 
  Verifying  : msgpack-3.1.0-4.el7.x86_64                                                                                                     8/16 
  Verifying  : groonga-plugin-suggest-11.0.4-1.el7.x86_64                                                                                     9/16 
  Verifying  : boost169-system-1.69.0-2.el7.x86_64                                                                                           10/16 
  Verifying  : groonga-libs-11.0.4-1.el7.x86_64                                                                                              11/16 
  Verifying  : zeromq3-3.2.5-1.el7.x86_64                                                                                                    12/16 
  Verifying  : xxhash-libs-0.8.0-3.el7.x86_64                                                                                                13/16 
  Verifying  : utf8proc-1.3.1-2.el7.x86_64                                                                                                   14/16 
  Verifying  : groonga-11.0.4-1.el7.x86_64                                                                                                   15/16 
  Verifying  : arrow-libs-4.0.0-1.el7.x86_64                                                                                                 16/16 

Installed:
  groonga.x86_64 0:11.0.4-1.el7                                                                                                                    

Dependency Installed:
  arrow-libs.x86_64 0:4.0.0-1.el7             boost169-filesystem.x86_64 0:1.69.0-2.el7             boost169-system.x86_64 0:1.69.0-2.el7         
  brotli.x86_64 0:1.0.7-5.el7                 gflags.x86_64 0:2.1.1-6.el7                           glog.x86_64 0:0.3.3-8.el7                     
  groonga-libs.x86_64 0:11.0.4-1.el7          groonga-plugin-suggest.x86_64 0:11.0.4-1.el7          libevent.x86_64 0:2.0.21-4.el7                
  libzstd.x86_64 0:1.5.0-1.el7                msgpack.x86_64 0:3.1.0-4.el7                          openpgm.x86_64 0:5.2.122-2.el7                
  utf8proc.x86_64 0:1.3.1-2.el7               xxhash-libs.x86_64 0:0.8.0-3.el7                      zeromq3.x86_64 0:3.2.5-1.el7                  

Complete!
```


If this PR is need, Groonga occurs error as a below point, but we success installing Groonga.

```
--> Processing Dependency: arrow-libs = 4.0.0 for package: groonga-libs-11.0.4-1.el7.x86_64
```

**Comment by kou** [comment]
OK.
I close this.

BTW, we need to update Apache Arrow to 4.0.1.

**Pull request closed without merging**